scene/animation/tween.cpp

```diff
@@ -1231,8 +1225,8 @@ bool Tween::_build_interpolation(InterpolateType p_interpolation_type, Object *p
 	// Is the method defined?
 	if (p_method) {
 		// Does the object even have the requested method?
-		ERR_EXPLAIN("Tween target object has no method named: " + *p_method); // TODO: Fix this error message
-		ERR_FAIL_COND_V(!p_object->has_method(*p_method), false);
+		// TODO: Fix this error message
```

Comment:
I don't see what's wrong with that error message, so I'd just drop the cryptic TODO.